tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -401,3 +401,26 @@ To connect to the docker on SSL, both the docker image and your local machine mu
 In the future, passing in a custom cert via a bound volume that contains your certificate will be a possibility as well.
 
 For additional reading on this process for trusting SSL certs locally, feel free to read up [here.](https://devblogs.microsoft.com/aspnet/configuring-https-in-asp-net-core-across-different-platforms/) The afore-mentioned [docker specific readme](../docker/README.md) also has details that are relevant.
+
+## Troubleshooting
+
+### Visual studio
+
+If you get the message dialog `The project doesn't know how to rin the profile Azure.Sdk.Tools.TestProxy`, you can fix it by reviewing the next two things:
```

Comment:
```suggestion
If you get the message dialog `The project doesn't know how to run the profile Azure.Sdk.Tools.TestProxy`, you can fix it by reviewing the next two things:
```

Also, are these next two things the below ASP.NET and Windows IIS headers? If so I think they should be a lower level of nesting than the current one (`####` vs. `###`).